docs/architecture/adr-028-public-key-addresses.md

```diff
@@ -0,0 +1,110 @@
+# ADR 028: Public Key Addresses
+
+## Changelog
+
+- 2020/08/18: Initial version
+
+## Status
+
+Proposed
+
```

Comment:
Let's add a `Summary` or `Abstract` section right after Changelog and Status.

Reply:
Could you make a proposal around this to the ADR template itself? I'm just trying to follow what's in the template.